dts/bindings/gpio/ti,am654-gpio-nexus.yaml

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2021, Yonatan Schachter
+# SPDX-License-Identifier: Apache-2.0
+
+description: GPIO controller for Davinci and keystone devices
+
+compatible: "ti,j721e-gpio-nexus"
```

Comment:
The Linux kernel already has:
```
.compatible = "ti,am654-gpio"
```

Why not reuse?

Reply:
ti,j721e-gpio-nexus is zephyr specific for having sub nodes for gpio in device tree, as the number of pins for a gpio controller is 32.

The compatible for the node is ti,am654-gpio-nexus and submodes are ti,am654-gpio.

Reply:
ti,j721e-gpio-nexus is zephyr specific for having sub nodes for gpio in device tree, as the number of pins for a gpio controller is 32.

The compatible for the node is ti,am654-gpio-nexus and submodes are ti,am654-gpio.